src/app/util/util.cpp

```diff
@@ -139,6 +139,7 @@ void MatterEnergyEvseModePluginServerInitCallback() {}
 void MatterPowerTopologyPluginServerInitCallback() {}
 void MatterElectricalEnergyMeasurementPluginServerInitCallback() {}
 void MatterElectricalPowerMeasurementPluginServerInitCallback() {}
+void MatterWaterHeaterManagementPluginServerInitCallback() {}
```

Comment:
Why do we have two implementations of this function?

Reply:
I will remove the one from water-heater-management-server.cpp